3rdParty/iresearch/core/formats/formats_10.cpp

```diff
@@ -1344,8 +1347,11 @@ void index_meta_reader::read(
 // --SECTION--                                              segment_meta_writer 
 // ----------------------------------------------------------------------------
 
+MSVC2015_ONLY(__pragma(warning(push)))
```

Comment:
Hm, at least with std::string we saw that this would actually be re-initialized on spawning new threads, and thus other threads could run into empty / half initialized strings; Did you try wether this doesn't occur here?

Reply:
are you saying static variables were initialized multiple times?

Reply:
yes - global `static const std::string foo("abc")` - we found out the hard way.

Reply:
none of the multi-threaded iresearch tests faced the issue you are describing